licenses/istio.io/client-go/LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2016-2022 Istio Authors
+   Copyright 2016-2020 Istio Authors
```

Comment:
Is changing 2022 to 2020 needed?

Reply:
Seems odd but it's auto generated. Needs to be fixed in client-go?

Reply:
Oops - I downgraded client-go accidentally. I'll fix that. Nice catch